app/modifiers/draw-chart.js

```diff
@@ -0,0 +1,69 @@
+import { registerDestructor } from '@ember/destroyable';
+import merge from 'deepmerge';
+import Modifier from 'ember-modifier';
+import Highcharts from 'highcharts/highstock';
+import highchartsAccessibilty from 'highcharts/modules/accessibility';
+
+const optionsForAllCharts = {
+  credits: {
+    enabled: false,
+  },
+
+  plotOptions: {
+    series: {
+      animation: false,
+    },
+  },
+
+  time: {
+    timezoneOffset: new Date().getTimezoneOffset(),
+  },
+};
+
+export default class DrawChartModifier extends Modifier {
+  constructor(owner, args) {
+    super(owner, args);
+
+    this.initializeHighcharts();
+  }
+
+  modify(element, [chart]) {
+    if (!chart) {
+      return;
+    }
+
+    this.drawChart({ chart, element });
+
+    registerDestructor(this, this.destroyChart.bind(this));
+  }
+
+  initializeHighcharts() {
+    highchartsAccessibilty(Highcharts);
+
+    this.highcharts = Highcharts;
+    this.highcharts.setOptions(optionsForAllCharts);
+  }
```

Comment:
Previously, we had dynamically imported Highcharts, then observed that the import is finished before rendering an svg element.

Although it's possible to (in a modifier) dynamically import Highcharts and its extra packages (like the accessibility module), I found that it's easier for others to debug and maintain code when we do static import.

Reply:
The accessibility module allows users to use their keyboard to interact with charts. We can also pass chart captions and ARIA labels to better describe axes.